Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx

```diff
@@ -48,6 +48,8 @@ MatrixOffsetTransformBase<TParametersValueType, VInputDimension, VOutputDimensio
   m_MatrixMTime.Modified();
   std::copy_n(offset.begin(), VOutputDimension, m_Translation.begin());
   this->ComputeMatrixParameters();
```

Comment:
Slightly off-topic, maybe, but the `this->ComputeMatrixParameters()` call does nothing at all: https://github.com/InsightSoftwareConsortium/ITK/blob/07f8459bdf913647dab52108d8ff284d05eb4fe4/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx#L678-L684

(I was expecting `ComputeMatrixParameters()` to interact with `m_FixedParameters`, but obviously, it does not!)

Reply:
If this `MatrixOffsetTransformBase(const MatrixType &, const OutputVectorType &)` constructor is really meant to be used, I guess it should do the same as the default-constructor, so it should then also do `m_InverseMatrixMTime = m_MatrixMTime`, just like the default-constructor:

https://github.com/InsightSoftwareConsortium/ITK/blob/fb4b463b38640c874202808311ff6e06afb1237e/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx#L30-L38

(But it does not need to call `ComputeMatrixParameters()` anyway.)